system/settings/settings.xml

```diff
@@ -564,6 +564,14 @@
           </constraints>
           <control type="list" format="string" />
         </setting>
+        <setting id="videoplayer.adjustresolution" type="boolean" label="13549" help="36163">
```

Comment:
I think of broken CRT Monitors :-) Remember the days when the CRT could do
1600x1200 but only at 85 hz and therefore one chose 1280x1024 at 100 hz?

2015-09-08 21:57 GMT+02:00 Rainer Hochecker <notifications@github.com>:

> In system/settings/settings.xml
> <https://github.com/FernetMenta/xbmc/pull/272#discussion_r38972063>:
>
> > @@ -564,6 +564,14 @@
> >            </constraints>
> >            <control type="list" format="string" />
> >          </setting>
> > +        <setting id="videoplayer.adjustresolution" type="boolean" label="13549" help="36163">
>
> can you think of any reason why you don't want to switch to 2160 if you
> have a tv that supports it and refresh rate matches?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FernetMenta/xbmc/pull/272/files#r38972063>.
>



-- 
                   Key-ID:     0x1A995A9B
                   keyserver: pgp.mit.edu
==============================================================
Fingerprint: 4606 DA19 EC2E 9A0B 0157  C81B DA07 CF63 1A99 5A9B